src/core.js

```diff
@@ -77,8 +66,19 @@ extend( QUnit, {
 
 	},
 
-	// DEPRECATED: QUnit.asyncTest() will be removed in QUnit 2.0.
-	asyncTest: asyncTest,
+	asyncTest: function() {
+		throw new Error(
+			"asyntTest is now removed from QUnit 2.0, check out our upgrade guide at " +
```

Comment:
s/asyntTest/asyncTest/. And as an aside, I notice that only some of the removals come with `throw`s... when will we fully complete the process and drop even these?